examples/js/loaders/GLTFLoader.js

```diff
@@ -1231,6 +1231,31 @@ THREE.GLTFLoader = ( function () {
 
 	}
 
+	function cloneBufferAttribute( attribute ) {
+
+		if ( attribute.isInterleavedBufferAttribute ) {
+
+			var count = attribute.count;
+			var itemSize = attribute.itemSize;
+			var array = attribute.array.slice( 0, count * itemSize );
+
+			for ( var i = 0; i < count; ++ i ) {
+
+				array[ i ] = attribute.getX( i );
+				if ( itemSize >= 2 ) array[ i + 1 ] = attribute.getY( i );
+				if ( itemSize >= 3 ) array[ i + 2 ] = attribute.getZ( i );
+				if ( itemSize >= 4 ) array[ i + 3 ] = attribute.getW( i );
```

Comment:
@donmccurdy Um, I'm not sure this section correct. If you use `i` in order to refer to the current array position, you are going to overwrite values after `array[ i ] = attribute.getX( i );` since you raise the variable in the `for` loop. I think it should look like this:

```js
var j = 0;

for ( var i = 0; i < count; ++ i ) {

    array[ j ++ ] = attribute.getX( i );
    if ( itemSize >= 2 ) array[ j ++ ] = attribute.getY( i );
    if ( itemSize >= 3 ) array[ j ++ ] = attribute.getZ( i );
    if ( itemSize >= 4 ) array[ j ++ ] = attribute.getW( i );

}
```

Reply:
I would change it directly. I've actually added this code section in #15413. I'm not sure how much effort it takes not enable copy/clone and serialization/deserialization of interleaved buffer attributes. For now, I vote to create "normal" buffer attributes if we have to clone or  serialization data.